typescript/packages/subsurface-viewer/src/SubsurfaceViewer.tsx

```diff
@@ -123,6 +123,11 @@ export interface SubsurfaceViewerProps {
     lights?: LightsType;
 
     children?: React.ReactNode;
+
+    /**
+     * Forces the component to use the records provided in 'layers' prop as is.
+     */
+    skipLayerPropsConversion?: boolean;
```

Comment:
Not happy with this name, any better proposals?

Reply:
jsObjectLayerPropsSupport ?

Reply:
or useTypedArrays? To underline the main purpose of the flag.

Reply:
Renamed.